### PR TITLE
Feature/add cookie page to footer

### PR DESCRIPTION
**Pull request by tahb**
# Context

[We need to display a cookie policy before we can be given the green light to go live.](https://dsdmoj.atlassian.net/wiki/spaces/NDSS/pages/3051684138/Cookies+policy)

# Changes in this PR

- add a static cookie page that informs the user of the one essential cookie we have that's used for authentication. [This allows us to avoid having a cookie banner](https://design-system.service.gov.uk/components/cookie-banner/#option-1-if-youre-only-using-essential-cookies) with accept/reject options that need to be remembered between page views
- the words for the page are copied from https://docs.google.com/document/d/13ZL3hHwZNw42ZxJdDtIvlbiCIorfaxlAaEAxuJF_X1A/edit#heading=h.btqgc1d9sfx0 and [Manage A Supervision](https://github.com/ministryofjustice/hmpps-manage-supervisions/pull/330)

I thought to introduce the concept of "static" as we will likely need other static pages in the footer such as accessibility and privacy pages. Hopefully this makes that easier to add on later.

## Screenshots of UI changes

Screenshot 2023-01-18 at 16-46-40 Cookies

# Release checklist

As part of our continuous deployment strategy we must ensure that this work is
ready to be released at any point. Before merging to `main` we must first
confirm:

## Pre merge checklist

- [ ] Have all changes to any dependencies been deployed to both `preprod` and
    `production` in a backwards compatible way? (This includes our API,
    infrastructure, third-party integrations and any secrets or environment variables)
- [ ] Has a required data migration been included in this change or been done in
    advance?

## Post merge checklist

Once we've merged we now need to release this through to production before
considering it done.

[Find the build-and-deploy job in CircleCI](https://app.circleci.com/pipelines/github/ministryofjustice/hmpps-temporary-accommodation-ui)
and work through the following steps:

- [ ] Has the product manager asked to provide approval for this feature?
  - [ ] Has the product manager approved?
- [ ] Manually approve release to preprod
- [ ] Verify change released to preprod
- [ ] Manually approve release to prod
- [ ] Verify change released to production

Should a release fail at any step, you as the author should now lead the work to
fix it as soon as possible.

### CODE REVIEW

File: server/controllers/staticController.test.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  const response: DeepMocked<Response> = createMock<Response>({})
+  const next: DeepMocked<NextFunction> = createMock<NextFunction>({})
+
+  let staticController: StaticController
+
+  beforeEach(() => {
+    staticController = new StaticController()
+  })
+
+  describe('index', () => {
```
Comment (bryangaledxw) [inline server/controllers/staticController.test.ts]: Think this should be `describe('cookies', () => {`

**Review by bryangaledxw — COMMENTED** [review]
Looks good!

Appreciate this is just the one page, but I think I'd still setup paths similar to how we're doing elsewhere - take a look at `paths/temporary-accommodation/manage.ts`. Maybe add a `static.ts` with the cookie path

Then in `nunjucksSetup.ts`, you'd want to change

```
 njkEnv.addGlobal('paths', {
    ...managePaths,
  })
```

to
```
 njkEnv.addGlobal('paths', {
    ...managePaths,
    ...staticPaths,
  })
```

to get access to those new paths in the template

File: server/routes/temporary-accommodation/index.ts [hunk 2]
```diff
@@ -11,11 +11,12 @@ import manageRoutes from './manage'
 export default function routes(controllers: Controllers, services: Services): Router {
   const router = Router()
 
-  const { dashboardController } = controllers
+  const { dashboardController, staticController } = controllers
 
   const { get } = actions(router, services.auditService)
 
   get('/', dashboardController.index(), { auditEvent: 'VIEW_DASHBOARD' })
+  get('/cookies', staticController.cookies(), { auditEvent: 'VIEW_COOKIES' })
```
Comment (tahb) [inline server/routes/temporary-accommodation/index.ts]: @bryangaledxw After the change to cookies to the `path` setup, should I be using it as a reference here too?

**Comment by tahb** [comment]
Thank you for the helpful feedback Bryan. I've added a commit and it's ready for your eyes again!

File: server/routes/temporary-accommodation/index.ts (hunk 2 above)
Comment (bryangaledxw) [reply server/routes/temporary-accommodation/index.ts]: Yep, so `/cookies` becomes `paths.static.cookies.pattern`

Comment (tahb) [reply server/routes/temporary-accommodation/index.ts]: Okay all done 👍 Hopefully included paths correctly, rather than calling it staticPaths.

Comment (bryangaledxw) [reply server/routes/temporary-accommodation/index.ts]: Cool, cheers, looks good. Strictly speaking I should have done he same for the dashboard!

**Review by bryangaledxw — APPROVED** [review]

**Pull request merged**